playbook-website/config/deploy/templates/shell.yaml.erb

```diff
@@ -0,0 +1,52 @@
+apiVersion: v1
+kind: PodTemplate
+metadata:
+  name: playbook-shell
+  labels:
+    app: playbook
+    app.kubernetes.io/component: shell
+template:
+  metadata:
+    generateName: playbook-shell-
+    labels:
+      app: playbook
+      app.kubernetes.io/component: shell
+  spec:
+    restartPolicy: Never
+    containers:
+      - name: task-runner
+        image: "image-registry.powerapp.cloud/playbook/playbook:<%= image_tag %>"
+        imagePullPolicy: IfNotPresent
+        command:
+          - /sbin/my_init
+        args:
+          - --
+          - sleep
+          - "10800"
+        env:
+          - name: PORT
+            value: "3000"
+          - name: SECRET_KEY_BASE
+            valueFrom:
+              secretKeyRef:
+                name: playbook
+                key: secret-key-base
+          - name: FORCE_SSL
+            value: "0"
+          - name: RACK_ENV
+            value: <%= appConfig["rackEnv"] %>
+          - name: RAILS_ENV
+            value: <%= appConfig["railsEnv"] %>
+          - name: RAILS_SERVE_STATIC_FILES
+            value: "true"
+          - name: RAILS_LOG_TO_STDOUT
+            value: "true"
+        resources:
+          limits:
+            cpu: 0.3
+            memory: 512Mi
+            ephemeral-storage: 100Mi
+          requests:
+            cpu: 0.1
+            memory: 512Mi
```

Comment:
Just noting that memory in review environments will be capped out with a single console pod.

```
Resource                    Used   Hard
--------                    ----   ----
configmaps                  1      10
limits.cpu                  600m   1
limits.ephemeral-storage    200Mi  300Mi
limits.memory               512Mi  1Gi
persistentvolumeclaims      0      0
pods                        2      4
replicationcontrollers      0      5
requests.cpu                200m   1
requests.ephemeral-storage  200Mi  300Mi
requests.memory             512Mi  1Gi
requests.storage            0      0
secrets                     6      10
services                    1      5
services.loadbalancers      0      0
```